tests/antithesis/test-template/robustness/finally/main.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2025 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build cgo && amd64
+
+package main
+
+import (
+	"flag"
+	"maps"
+	"slices"
+	"time"
+
+	"github.com/anishathalye/porcupine"
+	"github.com/antithesishq/antithesis-sdk-go/assert"
+	"go.uber.org/zap"
+
+	"go.etcd.io/etcd/tests/v3/antithesis/test-template/robustness/common"
+	"go.etcd.io/etcd/tests/v3/robustness/report"
+	"go.etcd.io/etcd/tests/v3/robustness/traffic"
+	"go.etcd.io/etcd/tests/v3/robustness/validate"
+)
+
+func main() {
+	local := flag.Bool("local", false, "run finally locally and connect to etcd instances via localhost")
+	flag.Parse()
+
+	reportPath, _, dirs := common.DefaultPaths()
+	if *local {
+		reportPath, _, dirs = common.LocalPaths()
+	}
+
+	lg, err := zap.NewProduction()
+	if err != nil {
+		panic(err)
+	}
+	r := report.TestReport{Logger: lg, ServersDataPath: dirs}
+	r.Client, err = report.LoadClientReports(reportPath)
+	assert.Always(err == nil, "Loaded client reports", map[string]any{"error": err})
+	defer func() {
+		if err := r.Report(reportPath); err != nil {
+			lg.Error("Failed to save validation report", zap.Error(err))
+		}
+	}()
+	testValidate(lg, r)
+}
+
+func testValidate(lg *zap.Logger, r report.TestReport) {
```

Comment:
```suggestion
func validate(lg *zap.Logger, r report.TestReport) {
```

Reply:
`validate` is already used for a package import.  I'll go with `doValidation` for now.